### ISSUE TITLE
Cross Platform Support For Cpp makefile (Unable to build on Windows)

### ISSUE THREAD

**Issue by duckkar** [nayuki/QR-Code-generator#42]
Hi, currently I am trying to use this library for generating qr code from my cpp program complied with mingw64 on windows. But on using mingw32-make I get these errors:
**mkdir -p .deps
touch .deps/timestamp
process_begin: CreateProcess(NULL, touch .deps/timestamp, ...) failed.
make (e=2): The system cannot find the file specified.
mingw32-make: *** [Makefile:82: .deps/timestamp] Error 2**
Please help.. I think its due makefile thats written for linux...
Moderators and Contributors to this project Please add support for cmake or any other cross platform utilities..
Thanks..

**Comment by nayuki** [comment]
You are right, the Makefile is intended for Linux and BSD. The support for both C/C++ development and makefiles on Windows has never been good. I have no idea what you are missing; it looks like your `mkdir` is broken or `touch` is broken.

Your options are the following:
* Compile the files directly on the command line: `g++ -std=c++11 QrCode.cpp QrSegment.cpp BitBuffer.cpp QrCodeGeneratorDemo.cpp`
* Import the source files into an IDE like Microsoft Visual Studio.
* Switch to developing on Linux (on a virtual machine, bare metal, server, etc.).

Don't bother addressing the "moderators and contributors to this project". It is all run by me. There are other closed issues and rejected pull requests with cmake files which you can browse at your own leisure.
